### PR TITLE
Add emoji feature

### PR DESCRIPTION
**Pull request by carolinar7**
Let me know what you think about the selection of emojis

### CODE REVIEW

File: src/main/java/com/google/sps/data/EmojiSelection.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+public final class EmojiSelection {
+
+  private static String[] emoji1 = {"👿","🖕","😾","😡","😠","😧","💔"};
```
Comment (gracieorozco) [inline src/main/java/com/google/sps/data/EmojiSelection.java]: I think it's important to keep this appropriate. Removing the first two emojis would be great.

**Review by gracieorozco — APPROVED** [review]
This pull request looks great. I've tested the branch and found no errors. After making the changes, the branch can be merged.

File: src/main/java/com/google/sps/data/EmojiSelection.java (hunk 1 above)
Comment (carolinar7) [reply src/main/java/com/google/sps/data/EmojiSelection.java]: You're right, I went ahead an removed it

**Comment by gracieorozco** [comment]
Thanks for making the changes, feel free to merge the branch.

**Review by gracieorozco — APPROVED** [review]

**Pull request merged**